build_tools/cmake/iree_lit_test.cmake

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 
 include(CMakeParseArguments)
+include(iree_installed_test)
```

Comment:
Want to also update https://github.com/google/iree/blob/main/build_tools/cmake/iree_check_test.cmake and https://github.com/google/iree/blob/main/build_tools/cmake/iree_run_binary_test.cmake ?

https://sourcegraph.com/search?q=repo:%5Egithub%5C.com/google/iree%24+file:iree_.*_test.cmake&patternType=regexp